deployment/CE04OSPS_Deploy.csv

```diff
@@ -21,7 +21,7 @@ TN-326,,,,CE04OSPS-PC01B-4B-PHSENA106,2,1,2015-08-03T05:41:00,2016-07-18T00:00:0
 TN-326,,,,CE04OSPS-PC01B-4D-PCO2WA105,2,1,2015-08-03T05:41:00,2016-07-18T00:00:00,ATAPL-69839-002-0105,,ATOSU-58336-00008,44.37415,-124.95648,,194,580,
 TN-326,,,,CE04OSPS-SF01B-2A-CTDPFA107,2,1,2015-08-03T02:21:00,2016-07-18T00:00:00,ATAPL-68870-002-0144,,ATOSU-66662-00013,44.37415,-124.95648,,,580,
 TN-326,,,,CE04OSPS-SF01B-2A-DOFSTA107,2,1,2015-08-03T02:21:00,2016-07-18T00:00:00,ATAPL-68870-002-0144,,ATOSU-58694-00007,44.37415,-124.95648,,,580,
-TN-326,,,,CE04OSPS-SF01B-2B-PHSENA108,2,1,2015-08-03T02:21:00,2016-07-18T00:00:00,ATAPL-68870-002-0144,,ATOSU-58337-00011,44.37415,-124.95648,,,580,
+TN-326,,,,CE04OSPS-SF01B-2D-PHSENA108,2,1,2015-08-03T02:21:00,2016-07-18T00:00:00,ATAPL-68870-002-0144,,ATOSU-58337-00011,44.37415,-124.95648,,,580,
```

Comment:
The reference designator is used as a primary key in the database to access information pertaining to this instrument. There over 18k data records for this instrument that will no longer be accessible if this reference designator is changed without first loading that information. Also, all the new information will still be written to the old reference designator until the port agent and instrument drivers have been updated. So - we need to have a coordinated effect to fix any reference designator for which data already exists.